dfn_configs.py

```diff
@@ -5,69 +5,77 @@
 
 from typing import Any, Callable, Optional, Tuple
 
-from entities.common import Angle, Circle, Diameter, Fill, GrabArea, Layer, Polygon, Position, Vertex, Width
+from entities.common import (
+    Angle,
+    Circle,
+    Diameter,
+    Fill,
+    GrabArea,
+    Layer,
+    Polygon,
+    Position,
+    Vertex,
+    Width,
+)
 from entities.package import Footprint
 
 # Maximal lead width as a function of pitch, Table 4 in the JEDEC
 # standard MO-229F, available (with registration!) from
 # https://www.jedec.org/system/files/docs/MO-229F.pdf
-LEAD_WIDTH = {
-    0.95: 0.45,
-    0.8: 0.35,
-    0.65: 0.35,
-    0.5: 0.30,
-    0.4: 0.25
-}
+LEAD_WIDTH = {0.95: 0.45, 0.8: 0.35, 0.65: 0.35, 0.5: 0.30, 0.4: 0.25}
```

Comment:
This was more readable before formatting, would a trailing comma revert the one-value-per-line formatting?

Reply:
Yes it does.